client/src/components/LeftDrawer/LeftDrawer.js

```diff
@@ -59,6 +67,18 @@ class LeftDrawer extends React.Component {
               </ListItem>
             </Link>
           ))}
+          <Link to={'/login'} key={4} className="link">
+            <ListItem button onClick={this.logout}>
+              <ListItemIcon>
+                <img
+                  alt="for pages"
+                  src="http://mawarupenguindrum.moonfruit.fr/communities/5/000/001/348/125/images/1860781.png"
+                  style={{ width: 30, height: 30 }}

```

Comment:
@zerfuth what is this photo LOL (but for real, if you want to include this, download it and put it in an assets folder and refer to that location)

Reply:
Yueh's fault :)